### DISCUSSION TITLE
When checking checkbox I get error in console in new Laravel Breeze installation

### DISCUSSION THREAD

**Discussion by ifkas** [Power-Components/livewire-powergrid#1279] [Q&A]
### Have you searched through other issues to see if your problem is already reported or has been fixed?

Yes, I did not find it.

### Did you read the documentation?

Yes, I did not find it.

### Have you tried to publish the views?

Yes - I didn't work.

### Is there an error in the console?

![1](https://github.com/Power-Components/livewire-powergrid/assets/987112/f5c7505a-05bf-4fd2-be02-e5f9adb31f18)
and
![2](https://github.com/Power-Components/livewire-powergrid/assets/987112/473e93ee-a46a-4dc7-94ae-3ec0217d2dd3)

### PHP Version

8.2.12

### PowerGrid

5.1.9lo

### Laravel

10.29.0

### Livewire

3.0.10

### Theme

Tailwind 3.x with tailwind/forms

### Describe the bug.

Hello everyone, I hope that someone can help or give me any direction regarding this, I have installed a fresh copy of the latest version of Laravel Breeze with Livewire (version  1.25.1), however, whenever I check one of the checkboxes I get this error in the console, in Firefox console "**Uncaught TypeError: window.Alpine.store(...) is undefined**" and in Chrome **Uncaught TypeError: Cannot read properties of undefined (reading 'add')>**

Laravel Breeze with Livewire comes bundled by default with Apline.js so I am not sure why this is happening, however, Alpine works if I try to put any Alpine code in that blade file. 

I am attaching the screenshots as a reference.

Am I missing something prior to installation, I have carefully read the docs. I was trying to create bulk action from [here](https://livewire-powergrid.com/table/bulk-actions.html#bulk-actions): 

P.S- I found something similar here [https://github.com/Power-Components/livewire-powergrid/issues/1116](https://github.com/Power-Components/livewire-powergrid/issues/1116) but I do not believe that's the case, that already is fixed but store global from Alpine is present.

Any type of help is appreciated, thank you.

### To Reproduce...

First click on "FOO" then....

### Extra information

```markdown
<?php
 //...
```

**Comment by philozofer** [comment]
I would also have the same issue. Not sure what causes this, thank you for any help

![image](https://github.com/Power-Components/livewire-powergrid/assets/9042494/2d4c833a-55c7-4edc-8fa7-cafb5e3d457c)

**Comment by junxiong96** [comment]
I also faced this issue, but I managed to solve it by importing the script into app.js and then importing it in the Blade using @vite:
```js
import './../../vendor/power-components/livewire-powergrid/dist/powergrid'
```

I'm not sure if this helps you.
